src/_data/side-nav.yml

```
@@ -1,8 +1,10 @@
-- title: Samples & tutorials
+- title: Tutorials
```

This should stay as **Tutorials & codelabs** for now since it's awkward having two layers of **Tutorials** > **Tutorials**, and reorganizing the pages underneath to make sense under **Tutorials** isn't in scope for this PR.
```suggestion
- title: Tutorials & codelabs
```

---

Also looks good to me after this change. Thanks @mit-mit!